PREREQUISITE.md

```diff
@@ -0,0 +1,248 @@
+
+&nbsp;&nbsp;&nbsp;&nbsp;2. Import the aar/jar file directly
+
+&nbsp;&nbsp;&nbsp;&nbsp;3. Import the Euphony module in your project
+
+### 1.1 Using Maven repository
+
+---
+
+1. Show **_Project window_** with **_Android project view_**. You can select a project view mode in the dropdown list, then open your `build.gradle` file under `Gradle Scripts`.
+
+<img width="392" alt="maven_001_auto_x2_colored_toned" src="https://user-images.githubusercontent.com/27720475/130187144-6bcd31e2-589e-48e0-91ff-d05d28bea4e4.png">
+
+2. Add the following line to the `dependencies` section:
+
+```gradle
+dependencies {
+	// other dependencies
+	// ...
+	implementation 'co.jbear.lib:euphony:0.7.1.6'
+}
+```
+
+### 1.2 Import the aar/jar file directly
+
+---
+
+#### Environment
+
+> OS : Windows 10  
+> Android Studio : Arctic Fox | 2020.3.1
+
+1. Download `euphony.aar` : [MavenCentral euphony artifact](https://search.maven.org/artifact/co.jbear.lib/euphony/0.7.1.6/aar) follow the link and download aar file
+
+<img width="392" alt="aar_001" src="https://user-images.githubusercontent.com/27720475/130188260-a0f3ee6d-7afc-4dfc-928e-5562eca476ba.png">
+
+2. Put `euphony.aar` file in `libs` folder, and just click `Refactor`
+
+<img width="392" alt="aar_002_auto_x2_colored_toned" src="https://user-images.githubusercontent.com/27720475/130187177-b97b55ef-158a-4975-b0f8-b9e8bfdc5886.png">
+
+<img width="392" alt="aar_003_auto_x2_colored_toned" src="https://user-images.githubusercontent.com/27720475/130187182-33d5f067-9165-4cc8-a8a0-04619d5dfdbe.png">
+
+3. Check your `build.gradle` in app module
+
+```gradle
+repositories {
+    flatDir {
+        dirs 'libs'
+    }
+}
+
+dependencies {
+    implementation name: 'euphony-0.7.1.6', ext: 'aar'
+}
+
+```
+
+then click `sync` and you can use euphony library!!
+
+### 1.3 Import the Euphony module in your project
+
+#### Environment
+
+> OS : Mac OS Big Sur  
+> Android Gradle Plugin Version : 4.2.1
+> Gradle Version : 6.7.1
+> Language : Java
+
+1. Download an zip file at [https://github.com/euphony-io/euphony](https://github.com/euphony-io/euphony) and unzip it in your computer
+
+<img width="392" alt="module_001" src="https://user-images.githubusercontent.com/27720475/130185343-fcff2b7b-a164-4b44-852e-ecd650dfaabf.png">
+
+2. Click `File` > `New` > `Import Module...`
+
+<img width="392" alt="module_002" src="https://user-images.githubusercontent.com/27720475/130185349-13eacf92-da1a-47cc-b2fa-cdc3c91b6aaf.png">
+
+<img width="392" alt="module_003" src="https://user-images.githubusercontent.com/27720475/130185355-dfef2041-53c8-4146-905f-39744e499a75.png">
+
+3. Put what you downloaded before in source directory and click `Finish`
+
+<img width="392" alt="module_004" src="https://user-images.githubusercontent.com/27720475/130185365-e4ef05af-60e1-4f37-b0a4-6b87cf2b17b1.png">
+
+<img width="392" alt="module_005" src="https://user-images.githubusercontent.com/27720475/130185374-b42d0860-6b2e-48a7-aa1f-a99eaf52435e.png">
```

Comment:
I'm sorry for late requirement, but I think it would better if you change this part like this. Because this is not an usual case after v0.8, but exception case.
we can reduce confusion by doing this:)
```suggestion
><img width="392" alt="module_005" src="https://user-images.githubusercontent.com/27720475/130185374-b42d0860-6b2e-48a7-aa1f-a99eaf52435e.png">
>
> If Could not get unknown property 'language' for build 'import_Euphony' of type org.gradle.invocation.DefaultGradle. error occur, please add a sentence below in setting.gradle(Project) and click 'Sync Now'
>```gradle
>gradle.ext.language = "java"; // or kotlin
>```
>
><img width="392" alt="module_006" src="https://user-images.githubusercontent.com/27720475/130185379-8f6fb9d8-d4a5-497e-b7ec-514aeb1088a0.png">
>
>> If you want to know more about 'Gradle', [https://www.baeldung.com/gradle-build-settings-properties](https://www.baeldung.com/gradle-build-settings-properties) will be helpful

><img width="392" alt="module_007" src="https://user-images.githubusercontent.com/27720475/130185391-02c82964-299e-484c-80e1-810254dd7070.png">
>
> If Caused by: groovy.lang.MissingPropertyException: Cannot get property 'signing.keyId' on extra properties extension as it does not exist error occur, please delete the sentence below in euphony/gradle.build and click 'Sync Now'apply from: file('publish.gradle')
```